.buildkite/pipeline_pr.py

```diff
@@ -85,16 +95,21 @@ def group(group_name, command, agent_tags=None, priority=0, timeout=30):
     agent_tags=["ag=1"],
 )
 
-pipeline = {
-    "agents": {"queue": "default"},
-    "steps": [
+steps = [step_style]
+changed_files = get_changed_files("main")
+if any(x.suffix != ".md" for x in changed_files):
+    steps += [
         step_style,
```

Comment:
It seems that `step_style` is added twice.

Reply:
True! Leftover from something I tried. Fixing.